uktrade/data-hub-frontend
# Update activity to be the default company view

## Change
Removes the `/companies/{id}/interactions` view and makes the `/companies/{id}/activity` view the default when viewing a company.

Code has been replaced with a `301` redirect just in case anyone has it bookmarked.

## Test instructions
1. Browse to `/companies/{id}` you should be redirected to `/companies/{id}/activity`
2. Browse to `/companies/{id}/interactions` you should be redirected to `/companies/{id}/activity`

**Checklist**
